libraries/src/Versioning/Versioning.php

```diff
@@ -34,11 +35,13 @@ class Versioning
 	public static function get($typeAlias, $id)
 	{
 		$db = Factory::getDbo();
+		$itemid = $typeAlias . '.' . $id;
 		$query = $db->getQuery(true);
 		$query->select($db->quoteName('h.version_note') . ',' . $db->quoteName('h.save_date') . ',' . $db->quoteName('u.name'))
 			->from($db->quoteName('#__history') . ' AS h ')
 			->leftJoin($db->quoteName('#__users') . ' AS u ON ' . $db->quoteName('u.id') . ' = ' . $db->quoteName('h.editor_user_id'))
```

Comment:
```suggestion
			->from($db->quoteName('#__history') . ' AS h ')
			->leftJoin($db->quoteName('#__users', 'u'), $db->quoteName('u.id') . ' = ' . $db->quoteName('h.editor_user_id'))
```
I think that's the proper syntax

Reply:
Your suggestion now conflicts xD can you make it again